src/Core/Context/TTC.idr

```diff
@@ -0,0 +1,32 @@
+module Core.Context.TTC
+
+import Core.Binary
+import Core.TTC
+import Core.Context
+
+%default covering
```

Comment:
Why just `covering`?

Reply:
Because that's what was in the original module.